### DISCUSSION TITLE
Dynamic Schema and static schema initialization at start up

### DISCUSSION THREAD

**Discussion by rhalim-dev** [Netflix/dgs-framework#1126] [Q&A]
Hey all,

I'm currently making a dynamic schema similar to https://netflix.github.io/dgs/advanced/dynamic-schemas/ from a metadata during start up. I also have a static schema (*.graphqls) that has various defined `Query` and `Mutations` that returns the type from the dynamic schema. It seems like the static schema is generated first which gives me the error 

```
There are problems with the GraphQL Schema:
* The field type 'Person' is not present when resolving type 'Mutation' [@10:1]
* The field type 'Person' is not present when resolving type 'Mutation' [@10:1]
* ....
```

How do I make the dynamic schema to be generated first? Or is there a way for both the static schema and dynamic schema to be combined/merged at runtime so that I won't get the above error?

**Comment by srinivasankavitha** [comment]
It is possible to merge static and dynamic types together. I just tried using a type defined in the dynamic schema in my static schema and verified it works properly. Could you share an example of a scenario that is not working for you?
